tests/integration/suite/test_system_project.py

```diff
@@ -93,12 +94,7 @@ def _check_system_sa_flag():
                 else:
                     return False
             else:
-                if sa.automount_service_account_token is None:
```

Comment:
I think we should leave this in, if it wasn't for the test we wouldn't have noticed fleet-system being missing

Reply:
you mean we keep the test as-is and error out on non-system ns if the flag is found to be false? But I think that seems wrong - if the namespace is not in system_ns, we should not care for the automount_service_account_token really.

Reply:
CIS scan tests will start failing for all the out-of-the-box namespaces that don't set this flag and that way we will know?

Reply:
Yeah that sounds good.